iOS_SDK/OneSignalSDK/Source/UIApplication+OneSignal.m

```diff
@@ -0,0 +1,55 @@
+/**
+ * Modified MIT License
+ *
+ * Copyright 2020 OneSignal
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * 1. The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * 2. All copies of substantial portions of the Software may only be used in connection
+ * with services provided by OneSignal.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#import "UIApplication+OneSignal.h"
+#import "OneSignalCommonDefines.h"
+
+@implementation UIApplication (OneSignal)
+
++ (BOOL)applicationIsActive {
+    if ([self isAppUsingUIScene]) {
+        if (@available(iOS 13.0, *)) {
+            
+            for (UIWindow *window in [UIApplication sharedApplication].windows) {
```

Comment:
We do this in a different way by using `UIApplication.sharedApplication.keyWindow.windowScene` instead.
https://github.com/OneSignal/OneSignal-iOS-SDK/blob/a0aa0e6fb482d649fdb781975c86cb24591d3239/iOS_SDK/OneSignalSDK/Source/OSMessagingController.m#L689-L698

See these notes as well
https://github.com/OneSignal/OneSignal-iOS-SDK/issues/648

Reply:
Fixed this!

Reply:
👍